tests/test_scripts.py

```diff
@@ -32,12 +32,8 @@ def test_script_generate_space_in_executable():
 def _read_launcher_data(section, kind):
     prefix = {"console": "t", "gui": "w"}[section]
     suffix = {"win-ia32": "32", "win-amd64": "64", "win-arm": "_arm"}[kind]
-    filename = os.path.join(
-        os.path.dirname(os.path.abspath(_scripts.__file__)),
-        f"{prefix}{suffix}.exe",
-    )
-    with open(filename, "rb") as f:
-        return f.read()
+    file = Path(_scripts.__file__).resolve().parent / f"{prefix}{suffix}.exe"
```

Comment:
This now hits the filesystem due to the `.resolve` which... I don't think that's strictly necessary?

Reply:
True, didn't notice it before. Removed `.resolve()`, things still work fine.